sandpack-react/src/components/CodeEditor/utils.ts

```diff
@@ -90,22 +91,29 @@ export const getSyntaxHighlight = (theme: SandpackTheme): HighlightStyle =>
       ...getSyntaxStyle(theme.syntax.static),
     },
     {
-      tag: tags.typeName,
+      tag: tags.tagName,
```

Comment:
Is this right? Shouldn't we keep the same?

Reply:
looks to have the same effect, tagName is more suggestive